scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -1034,19 +1048,31 @@ final int getNextReceiveMessageSequenceNumber() {
 	}
 
 	public void addApplicationDataForDeferredProcessing(RawData outgoingMessage) {
-		// backwards compatibility, allow on outgoing message to be stored.
-		int max = maxDeferredProcessedApplicationDataMessages == 0 ? 1 : maxDeferredProcessedApplicationDataMessages;
+		// backwards compatibility, allow one outgoing message to be stored.
+		int max = maxDeferredProcessedOutgoingApplicationDataMessages == 0 ? 1 : maxDeferredProcessedOutgoingApplicationDataMessages;
```

Comment:
as there is a default value > 0, do we need this backwards compatibility stuff ?

Reply:
OK